File: program1.c
#include<stdio.h>
int main(){
    int n,s=0;
    printf("enter number");
    scanf("%d",&n);
    for(int i=1;i<=n;i++)
    {
        s=s+i;
    }
    printf("sum=%d",s);
    return 0;
}

File: Day-11/program1.java
import java.util.Scanner;
public class program1
{//opening class
    static int sum(int a, int b) //FUNCTION TO CALCULATE SUM
{
        return a + b;
    }
    public static void main(String[] args) 
    {//opening main
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter first number: ");
        int num1 = sc.nextInt();
        System.out.print("Enter second number: ");
        int num2 = sc.nextInt();
        int r= sum(num1, num2);//calling function
        System.out.println("Sum = " + r);
        sc.close();
    }//closing main
}//opening class
